pkg/processor/procbuiltin/unwrap.go

```diff
@@ -232,6 +232,11 @@ func (o *openCDCUnwrapper) UnwrapPayload(structData record.StructuredData) (reco
 				Before: nil,
 				After:  convertedData,
 			}
+		case nil: // This can take place in operations such as `delete`
```

Comment:
I believe this will solve the problem in most cases. There are a few additional cases we need to think about:
* updates can have a before field
* deletes can have a before field

In other words, if `after` is `nil`, that doesn't necessarily mean it's a delete operation.

As an alternative, maybe we can do something like:
```go
payload = record.Change{
    Before: convertData(p["before"])
    After: convertData(p["after"])
}
```
```go
func convertData(data interface{}) {
    // check if data is nil or a string or map
}
```